### DISCUSSION TITLE
Role and permission on installation process

### DISCUSSION THREAD

**Discussion by schir94** [pimcore/pimcore#15004] [Q&A / Support]
Hello, 
When installing pimcore there is any .yaml config file (like workflow) in which we can define user role and permission? Because right now every time when installing, we have to redefine permissions and access levels. 
In the related section of the docs i couldn't find the answer

Thanks in advance

**Comment by kingjia90** [comment]
Unfortunately no, AFAIK role and permissions are all managed in the database
